lib/hammer_cli_import/contenthost.rb

```diff
@@ -93,7 +93,7 @@ def import_single_row(data)
         @map << {
           :org_id => data['organization_id'].to_i,
           :host_id => data['server_id'].to_i,
-          :uuid => c_host['uuid']}
+          :uuid => c_host['id']}
```

Comment:
Won't this still need to be the UUID in Candlepin in order for the transitioning systems to be able to register back? Trying to recall my understanding of how this works, but I thought it created a host object in Foreman with a subscription facet thus giving it a Candlepin UUID. On the real system, it de-registers from Sat 5, and then registers to Foreman passing along the Candlepin UUID to link the system to the pre-created host record.